pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -151,7 +151,7 @@ stdenv.mkDerivation (finalAttrs: {
   ] ++ lib.optionals stdenv.isDarwin [
     libedit
     readline
-  ] ++ lib.optional (stdenv.isDarwin && !stdenv.isAarch64) (
+  ] ++ lib.optional (stdenv.isDarwin && apple_sdk ? sdk) (
```

Comment:
Is this still necessary? `libproc.h` is included in the 10.12 SDK Libsystem and should be identical to this one.

Reply:
hmm, webkitgtk is marked as broken on darwin so not sure. i can add a comment stating it might not be necessary.